gems/spree_auth_devise/GHSA-gpqc-4pp7-5954.yml

```diff
@@ -0,0 +1,33 @@
+---
+gem: spree_auth_devise
+ghsa: gpqc-4pp7-5954
+url: https://github.com/solidusio/solidus_auth_devise/security/advisories/GHSA-xm34-v85h-9pg2
```

Comment:
This advisory is for `solidus_auth_devise`, but the gem name is `spree_auth_devise`.

Reply:
Solidus is a popular fork of Spree, so they share a lot of dependencies

Reply:
This file will be removed though as it duplicates https://github.com/rubysec/ruby-advisory-db/blob/master/gems/spree_auth_devise/CVE-2021-41275.yml

Reply:
If `solidus_auth_devise` is also affected, feel free to submit a correction to the GHSA to also list the `solidus_auth_devise` gem package as being affected. Then we could safely add an entry for it in `gems/solidus_auth_devise/`.

Reply:
@postmodern there is a separate GHSA for `solidus_auth_devise`: https://github.com/solidusio/solidus_auth_devise/security/advisories/GHSA-xm34-v85h-9pg2
It also has a separate CVE entry: https://nvd.nist.gov/vuln/detail/CVE-2021-41274
And that entry is already be included in `ruby-advisory-db`: https://github.com/rubysec/ruby-advisory-db/blob/59565b8df4010dbc343e6d0cf055ef2339cbad89/gems/solidus_auth_devise/CVE-2021-41274.yml
Thus this file seems to be redundant.